i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hdfs.md

```diff
@@ -50,73 +49,21 @@ CREATE TABLE hdfs_load_node (
-          <th class="text-left" style={{width: '7%'}}>默认值</th>
-          <th class="text-left" style={{width: '10%'}}>数据类型</th>
-          <th class="text-left" style={{width: '65%'}}>描述</th>
-      </tr>
-    </thead>
-    <tbody>
-    <tr>
-        <td><h5>sink.rolling-policy.file-size</h5></td>
-        <td style={{wordWrap: 'break-word'}}>128MB</td>
-        <td>MemorySize</td>
-        <td>滚动前 part 文件的最大值。</td>
-    </tr>
-    <tr>
-      <td><h5>sink.rolling-policy.rollover-interval</h5></td>
-      <td style={{wordWrap: 'break-word'}}>30 min</td>
-      <td>String</td>
-      <td>滚动前，part 文件处于打开状态的最大时长（默认值30分钟，以避免产生大量小文件）。
-       检查频率是由 'sink.rolling-policy.check-interval' 属性控制的。</td>
-    </tr>
-    <tr>
-      <td><h5>sink.rolling-policy.check-interval</h5></td>
-      <td style={{wordWrap: 'break-word'}}>1 min</td>
-      <td>String</td>
-      <td>基于时间的滚动策略的检查间隔。
-      该属性控制了基于 'sink.rolling-policy.rollover-interval' 属性检查文件是否该被滚动的检查频率。</td>
-    </tr>
-    </tbody>
-</table>
+| 参数  | 默认值 | 数据类型 | 描述  |
+| --- | --- | --- | --- |
+|  sink.rolling-policy.file-size | 128MB | MemorySize | 滚动前 part 文件的最大值。 |
+|  sink.rolling-policy.rollover-interval | 30 min | String | 滚动前，part 文件处于打开状态的最大时长（默认值30分钟，以避免产生大量小文件）。 检查频率是由 'sink.rolling-policy.check-interval' 属性控制的。 |
+|  sink.rolling-policy.check-interval | 1 min | String | 基于时间的滚动策略的检查间隔。 该属性控制了基于 'sink.rolling-policy.rollover-interval' 属性检查文件是否该被滚动的检查频率。 |
 
 #### 文件合并 
+
 支持文件能力，允许在较小的 checkpoint 下不产生大量的小文件。
-<table class="table table-bordered">
-    <thead>
-      <tr>
-          <th class="text-left" style={{width: '10%'}}>参数</th>
-          <th class="text-left" style={{width: '7%'}}>默认值</th>
-          <th class="text-left" style={{width: '10%'}}>数据类型</th>
-          <th class="text-left" style={{width: '65%'}}>描述</th>
-      </tr>
-    </thead>
-    <tbody>
-    <tr>
-        <td><h5>auto-compaction</h5></td>
-        <td style={{wordWrap: 'break-word'}}>false</td>
-        <td>Boolean</td>
-        <td>在流式 sink 中是否开启自动合并功能，数据首先会被写入临时文件。
-        当 checkpoint 完成后，该检查点产生的临时文件会被合并，这些临时文件在合并前不可见。</td>
-    </tr>
-    <tr>
-      <td><h5>compaction.file-size</h5></td>
-      <td style={{wordWrap: 'break-word'}}>(none)</td>
-      <td>String</td>
-      <td>合并目标文件大小，默认值为滚动文件大小。</td>
-    </tr>
-    <tr>
-      <td>inlong.metric.labels</td>
-      <td>可选</td>
-      <td style={{wordWrap: 'break-word'}}>(none)</td>
-      <td>String</td>
-      <td>inlong metric 的标签值，该值的构成为groupId=[groupId]&streamId=[streamId]&nodeId=[nodeId]。</td> 
-    </tr>
-    </tbody>
-</table>
+
+| 参数  | 默认值 | 数据类型 | 描述  |
+| --- | --- | --- | --- | 
+|  auto-compaction | false | Boolean | 在流式 sink 中是否开启自动合并功能，数据首先会被写入临时文件。 当 checkpoint 完成后，该检查点产生的临时文件会被合并，这些临时文件在合并前不可见。 |
+|  compaction.file-size | (none) | String | 合并目标文件大小，默认值为滚动文件大小。 |
+| inlong.metric.labels | (none) | String | inlong metric 的标签值，该值的构成为groupId=g[groupId]&streamId=[streamId]&nodeId=[nodeId]。 |
```

Comment:
Add a space for `groupId...` between chinese words.

Reply:
Done.